lib/eex/lib/eex/tokenizer.ex

```diff
@@ -154,6 +158,13 @@ defmodule EEx.Tokenizer do
     :expr
   end
 
+  # Tokenize the remaining passing check_terminators as false,
+  # which relax the tokenizer to not error on unmatched pairs.
+  # If the tokens start with an "end" we have a middle expr.
```

Comment:
@josevalim I would remove the last comment line (`If the tokens start with an "end" we have a middle expr.`) or move it before the pattern matching (although it's a bit redundant).

Reply:
I would say it is still relevant info for the callers of this function. :)